sdv/cag/base.py

```diff
@@ -174,14 +174,23 @@ def reverse_transform(self, data):
         """
         data = self._convert_data_to_dictionary(data, self.metadata, copy=True)
         reverse_transformed = self._reverse_transform(data)
+        missing_columns = []
         for table_name, table in reverse_transformed.items():
+            required_columns = self._original_data_columns[table_name]
+            # Temporarily add missing columns not sampled yet
+            for col in required_columns:
+                if col not in table.columns:
+                    table[col] = 0
+                    missing_columns.append(col)
```

Comment:
why do we need to add this if we're just removing it after trying to cast? Can't we just skip those columns when casting?

Reply:
Yes done in [202e8fb](https://github.com/sdv-dev/SDV/pull/2531/commits/202e8fb25c0dcf53d7c97c1198b2f0109d35e5b5)